src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,101 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final String ARRAY_OUT_OF_BOUND_MESSAGE = "This index is out "
+            + "of the array range...";
+    private static final String NO_SUCH_ELEMENT_MESSAGE = "There is no such element...";
+    private static final int DEFAULT_CAPACITY = 10;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 is the default value of int